### PR TITLE
Connected Scrapers to DB

### PR DESCRIPTION
**Pull request by Meggie-H**

### CODE REVIEW

File: src/delete.ts [hunk 1]
```diff
@@ -180,3 +35,58 @@ import ProductInterface from "./models/models";
 
 	await browser.close();
 })();
+
+async function getProducts() {
+	return fetch(
+		`${process.env.ELASTIC_URL}/products/_search` ||
+			"https://localhost:9200/products/_search",
+		{
+			method: "post",
```
Comment (AyaazStreet) [inline src/delete.ts:44]: Should this getProducts not be a get method instead of post?

**Review by AyaazStreet — CHANGES_REQUESTED** [review]

File: src/delete.ts (hunk 1 above)
Comment (Meggie-H) [reply src/delete.ts:44]: No, it needs a body

**Review by AyaazStreet — APPROVED** [review]
Looks good

File: src/index.ts [hunk 2]
```diff
@@ -83,12 +91,146 @@ import ProductInterface from "./models/models";
...
+					return resp.json();
+				})
+				.then((res) => console.log(res))
+				.catch((err) => console.error(err.message));
+		}
 	} catch (error) {
 		console.error("Error while scraping:", error);
 	} finally {
 		await browser.close();
 		console.log("\nWeb scraping ended X_X\n");
```
Comment (Musa-Mabasa) [inline src/index.ts:152]: Console logs

File: src/index.ts [hunk 3]
```diff
@@ -83,12 +91,146 @@ import ProductInterface from "./models/models";
...
+
+			if (!createResponse.ok) {
+				throw new Error(
+					`Failed to create index! status: ${createResponse.status}`
+				);
+			}
+
+			console.log("Products index created successfully.");
+		} else {
+			console.log("Products index already exists.");
```
Comment (Musa-Mabasa) [inline src/index.ts:231]: Some more console logs

**Review by Musa-Mabasa — APPROVED** [review]
Everything looks good, just left some comments about console logs

**Pull request merged**